src/App.js

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
-
 import './App.css';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './Components/PostList/PostList';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
+const getCommentPostUsers = (comment, post, user) => {
+  const newPost = post.map(item => (
+    {
+      ...item,
+      user: user.filter(a => a.id === item.userId),
```

Comment:
```suggestion
      user: user.find(a => a.id === item.userId),
```
Такой пользователь всего один